### PR TITLE
Reference this guide first, and notify of deviations

### PR DESCRIPTION
**Pull request by bergerhoffer**
Per #14

Preview: http://file.rdu.redhat.com/~ahoffer/2020/style_guide/pr-20/master.html#_about_this_guide

### CODE REVIEW

**Comment by bergerhoffer** [comment]
Preview: http://file.rdu.redhat.com/~ahoffer/2020/style_guide/pr-20/master.html#_about_this_guide

**Comment by bergerhoffer** [comment]
@redhat-documentation/ccs-style-council For review please

**Comment by gregraka** [comment]
Hi Andrea (@bergerhoffer),

Your addition of "Reference this guide first, and notify of deviations," looks good!

Thanks,
--Greg

File: supplementary_style_guide/introduction/about-guide.adoc [hunk 1]
```diff
@@ -3,8 +3,8 @@
...
-
-// TODO: Mention that this should be referenced before the ibm style guide as it overrides some of the existing guidance
+This guide provides style guidelines and preferred term usage for link:https://www.redhat.com/[Red Hat] product and cross-product solution documentation. It is intended to be used as a supplement to _The IBM Style Guide_, which is the primary source of guidance for Red Hat product documentation.
 
 This guide was created by the Red Hat Customer Content Services team, who is responsible for producing product documentation for Red Hat customers. It may also be used by upstream communities who want to align more closely with the standards used by Red Hat product documentation.
+
+When looking for style guidance, reference this guide first, because it overrides certain guidance from _The IBM Style Guide_. If you must deviate from the guidance in either guide, notify the style council by opening an link:https://github.com/redhat-documentation/doc-style/issues[issue]. This way, the deviation can be discussed by the style council and included in this guide.
```
Comment (gregraka) [inline supplementary_style_guide/introduction/about-guide.adoc:10]: Andrea (@bergerhoffer), LGTM. Thanks!

**Review by gregraka — APPROVED** [review]
LGTM

**Review by gregraka — APPROVED** [review]
LGTM

**Comment by bergerhoffer** [comment]
@redhat-documentation/ccs-style-council Can I get a review/approval on this PR please?

**Comment by cbudz** [comment]
@bergerhoffer I'm on it.

**Review by cbudz — APPROVED** [review]
Everything looks good here to me.

**Pull request merged**